Sources/mpc-core-kit-swift/core/Web3AuthOptions.swift

```diff
@@ -0,0 +1,24 @@
+import FetchNodeDetails
+import Foundation
+
+public class CoreKitWeb3AuthOptions {
+    public let web3AuthClientId: String
+    public let manualSync: Bool
+    public let web3AuthNetwork: Web3AuthNetwork
+    public let storageKey: String?
+    public let sessionTime: Int?
+    public let disableHashFactor: Bool
```

Comment:
would be good to keep arg names same as web , for the sake of docs consistency
disableHashedFactorKey => disableHashedFactorKey
localStorage => storage

Also it seems like we are missing `hashedFactorNonce` param.

Reply:
@AyushBherwani1998

Reply:
Done